src/code/z_quake.c

```diff
@@ -131,43 +170,43 @@ s16 Quake_GetFreeIndex(void) {
     return ret;
 }
 
-QuakeRequest* Quake_AddImpl(Camera* camera, u32 callbackIdx) {
-    s16 idx = Quake_GetFreeIndex();
-    QuakeRequest* req = &sQuakeRequest[idx];
+QuakeRequest* Quake_AddImpl(Camera* camera, u32 type) {
+    s16 index = Quake_GetFreeIndex();
+    QuakeRequest* req = &sQuakeRequest[index];
 
     __osMemset(req, 0, sizeof(QuakeRequest));
     req->camera = camera;
     req->camId = camera->camId;
-    req->callbackIdx = callbackIdx;
+    req->type = type;
     req->isShakePerpendicular = true;
-    req->randIdx = ((s16)(Rand_ZeroOne() * (f32)0x10000) & ~3) + idx;
+    req->randIndex = ((s16)(Rand_ZeroOne() * (f32)0x10000) & ~3) + index;
```

Comment:
`Rand_ZeroOne` returns a float, so the 0x10000 will automatically get converted to a float, so the cast is unnecessary
```suggestion
    req->randIndex = ((s16)(Rand_ZeroOne() * 0x10000) & ~3) + index;
```